tiup/tiup-playground.md

```diff
@@ -118,18 +108,24 @@ tiup playground --db.binpath /xx/tidb-server
 
 By default, only one instance is started for each TiDB, TiKV, and PD component. To start multiple instances for each component, add the following flag:
 
-{{< copyable "shell-regular" >}}
-
 ```shell
 tiup playground --db 3 --pd 3 --kv 3
 ```
 
+### Specify a tag when starting the TiDB cluster
+
+After you stop a TiDB cluster started using TiUP playground, all cluster data is cleaned up as well. To start a TiDB cluster using TiUP playground and ensure that the cluster data is not cleaned up automatically, you can specify a tag when starting the cluster. After specifying the tag, you can find the cluster data in the `~/.tiup/data` directory. Run the following command to specify a tag:
+
+```shell
+tiup playground --tag <tagname>
+```
+
+For clusters started in this way, the data files are retained after the cluster is stopped. You can use this tag to start the cluster next time so that you can use the data kept since the cluster is stopped.
```

Comment:
```suggestion
For clusters started in this way, the data files are retained after the cluster is stopped. You can use this tag to start the cluster next time so that you can use the data kept since the cluster was stopped.
```